sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2202,6 +2202,40 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
       .integerConf
       .createWithDefault(1024)
 
+  /**
+   * refer to dev doc: `replay-exec.md`
+   * only supports "project" now
+   */
+  val TEST_REPLAY_EXEC_TYPE =
```

Comment:
same comment as above I would rather see these called like DEBUG instead of test.

Reply:
Done.